src/arch/xtensa/configs/cannonlake_defconfig

```diff
@@ -3,6 +3,7 @@ CONFIG_CAVS_DMIC=y
 CONFIG_CAVS_SSP=y
 CONFIG_CAVS_ALH=y
 CONFIG_CAVS_LPS=y
+CONFIG_CAVS_LPRO=y
```

Comment:
Can you change this line also:
https://github.com/thesofproject/sof/blob/e9bec4a559208ee8ae6b44ac24bcd8943bd0f4ee/src/platform/Kconfig#L224
It's better to have defaults in defconfigs than in Kconfig files